### PR TITLE
Updating the documentation

### PR DESCRIPTION
**Pull request by MariusNi**
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-osconfig/blob/main/CONTRIBUTING.md).
- [X] All unit tests are passing.
- [X] I have merged the latest `main` branch prior to this PR submission.
- [X] I submitted this PR against the `main` branch.

### CODE REVIEW

**Review by robertschaedler3 — APPROVED** [review]
Awesome! 👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,25 +1,14 @@
 # Introduction
 
-Azure Device OS Configuration (OSConfig) is a modular services stack running on a Linux IoT Edge device that facilitates remote device management over Azure as well from local management authorities. OSConfig contains a PnP Agent, a Management Platform (Modules Manager) and several Management Modules. For more information on OSConfig see [OSConfig North Star Architecture](docs/architecture.md) and [OSConfig Roadmap](docs/roadmap.md).
+Azure Device OS Configuration (OSConfig) is a modular configuration stack for�Linux Edge devices. OSConfig supports multi-authority device management over Azure and Azure Portal/CLI (via Azure PnP, IoT Hub, Azure Policy), GitOps, as well as local management (such as from OOBE). For more information on OSConfig see [OSConfig North Star Architecture](docs/architecture.md).
```
Comment (AhmedBM) [inline CONTRIBUTING.md:3]: I would recommend use Out of Box Experience (OOBE) instead of just OOBE

**Review by AhmedBM — DISMISSED** [review]
Looks good, just a little comment on expanding acronyms.

File: CONTRIBUTING.md (hunk 1 above)
Comment (MariusNi) [reply CONTRIBUTING.md]: Agree. Please review the update.

**Review by AhmedBM — APPROVED** [review]
👌

**Review by robertschaedler3 — APPROVED** [review]

**Pull request merged**